meta-balena-common/recipes-core/initrdscripts/files/watchdog

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+watchdog_enabled() {
+    [ -c /dev/watchdog ] && return 0 || return 1
+}
+
+watchdog_run() {
+    watchdog -t 10 -T 60 -F /dev/watchdog &
```

Comment:
Where is this `watchdog` binary coming from in the initramfs?

Reply:
It's from Busybox. You can see its documentation from here: https://busybox.net/downloads/BusyBox.html